draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -3079,6 +3169,108 @@ curves including Montgomery and twisted Edwards curves.
 Tibouchi {{T14}} and Aranha et al. {{AFQTZ14}} generalize these results.
 This document does not deal with this complementary problem.
 
+# An alternative hash\_to\_field function {#appx-hashtofield-alt}
```

Comment:
I'm less thrilled about adding this. What's the driving use case?

Reply:
See #284 --- basically, the alternative method is much better for high-degree extension fields of small-ish characteristic, like [curve9767](https://github.com/pornin/curve9767), which is nice for microcontrollers.